src/order/router.js

```diff
@@ -0,0 +1,36 @@
+const { Router } = require("express");
+const GetOrdersRequestSchema = require("./schema");
+const { GetOrdersRequest } = require("./models");
+
+const router = Router();
+
+module.exports = router;
+
+const signer = require("../helpers/signer.service").init(process.env.KEY);
+const orderService = require("./service").init(process.env.REPORTING_API_URL);
+
+const secretsService = {
+  reportingApiUrl: () => process.env.REPORTING_API_URL,
+  key: () => process.env.KEY,
+  publisherToken: () => process.env.PUBLISHER_TOKEN,
+};
+
+router.post("/", async (req, res) => {
+  const { error } = GetOrdersRequestSchema.validate(req.body);
```

Comment:
why are we making it that complicated?
if we want to show examples of how to use orders API / assets upload, wouldn't it make more sense to put examples with plain objects instead of doing double validations on the input and letting them figure out the request body from the Joi validation?

Reply:
it was written by @shaiboujuappcharge a long time ago, I didn't touch this. I can remove this to make it easier to understand